[13.0][IMP] stock_demand_estimate: enhance views

LoisRForgeFlow opened the pull request:
* Add a graph view.
* Add handy filters: (not) expired; group by date, location and/or product.
* Relabel `product_qty` as "Real Quantity" was really confusing.

@ForgeFlow

LoisRForgeFlow committed:
[IMP] stock_demand_estimate: enhance views

* Add a graph view.
* Add handy filters: (not) expired; group by date, location and/or product.
* Relabel `product_qty` as "Real Quantity" was really confusing.

rousseldenis reviewed (approved):
Code and functional review

LoisRForgeFlow committed:
[FIX] stock_demand_estimate_matrix: remove action extension

no need to modify action, once the tree view is set as editable it
will work as itended, no need to remove form from the view modes.

BernatPForgeFlow reviewed (approved):
LGTM!

LoisRForgeFlow commented:
/ocabot merge minor
